monai/transforms/compose.py

```diff
@@ -148,26 +73,23 @@ def execute_compose(
         start: the index of the first transform to be executed. If not set, this defaults to 0
         end: the index after the last transform to be exectued. If set, the transform at index-1
             is the last transform that is executed. If this is not set, it defaults to len(transforms)
-        lazy_evaluation: whether to enable lazy evaluation for lazy transforms. If False, transforms will be
+        lazy: whether to enable :ref:`lazy evaluation<Lazy Resampling>` for lazy transforms. If False, transforms will be
             carried out on a transform by transform basis. If True, all lazy transforms will
             be executed by accumulating changes and resampling as few times as possible.
-            A `monai.transforms.Identity[D]` transform in the pipeline will trigger the evaluation of
-            the pending operations and make the primary data up-to-date.
         overrides: this optional parameter allows you to specify a dictionary of parameters that should be overridden
             when executing a pipeline. These each parameter that is compatible with a given transform is then applied
-            to that transform before it is executed. Note that overrides are currently only applied when lazy_evaluation
-            is True. If lazy_evaluation is False they are ignored.
-            currently supported args are:
-            {``"mode"``, ``"padding_mode"``, ``"dtype"``, ``"align_corners"``, ``"resample_mode"``, ``device``},
-            please see also :py:func:`monai.transforms.lazy.apply_transforms` for more details.
-        override_keys: this optional parameter specifies the keys to which ``overrides`` are to be applied. If
-            ``overrides`` is set, ``override_keys`` must also be set.
+            to that transform before it is executed. Note that overrides are currently only applied when
+            :ref:`lazy resampling<Lazy Resampling>` is enabled for the pipeline or a given transform. If lazy is False
+            they are ignored. Currently supported args are:
+            {``"mode"``, ``"padding_mode"``, ``"dtype"``, ``"align_corners"``, ``"resample_mode"``, ``device``}.
         threading: whether executing is happening in a threaded environment. If set, copies are made
             of transforms that have the ``RandomizedTrait`` interface.
-        log_stats: whether to log the detailed information of data and applied transform when error happened,
-            for NumPy array and PyTorch Tensor, log the data shape and value range,
-            for other metadata, log the values directly. default to `False`.
-        verbose: whether to print debugging info when lazy_evaluation=True.
+        log_stats: log errors when they occur in the processing pipeline. By default, this is set to False, which
```

Comment:
I reintroduced `log_stats` because I thought it had been added for 1.2 but when I double checked, it was there in 1.1. As such, we can't just remove it. I'd advocate for deprecating it and folding its into `logger_name` after a couple of releases have gone by. I could put a deprecation warning in for it now if we think that is a good idea.

Reply:
If so, can we just merge the expected behavior of `log_stats` and `logger_name` and still use `log_stats`?

Thanks.

Reply:
How about we deprecate `log_stats`, and point people at `logger_name`? It is more consistent with the rest of the API

Reply:
We'll keep `log_stats`

Reply:
Done